apps/web/src/lib/db.ts

```diff
@@ -1,9 +1,29 @@
 import pg from "pg";
 const { Pool, types } = pg;
 
-const pool = new Pool({
+// By default we assume that the `DATABASE_URL` connection string parsing is sufficient.
+// It fails, however, when pointed out at URL for a managed Postgres db hosted by DigitalOcean.
+//
+// Therefore let's provide an opt-in method of providing a CA for the cert info.
+// In order to use a DO managed DB URL, you must prune the `?sslmode=require` from the connection string!
+// This is documented here: https://node-postgres.com/features/ssl#usage-with-connectionstring
+const dbConfigTemp = {
   connectionString: process.env.DATABASE_URL,
-});
+  ssl: {},
```

Comment:
In js/ts you can do in-place conditionals in objects like this:

```suggestion
  ...(process.env.CA_CERT && {
    ssl: {
        rejectUnauthorized: true,
        ca: process.env.CA_CERT,
      },
  }),
```

Reply:
Just a style preference, your code works perfectly fine as is.

Reply:
TIL! Great suggestion, added. (I also tested with a new ad-hoc container build to confirm I didn't break anything with my clumsy TS.)